RadarSDK/RadarAPIClient.m

```diff
@@ -269,9 +274,56 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
                         NSArray<RadarEvent *> *events = [RadarEvent eventsFromObject:eventsObj];
                         RadarUser *user = [[RadarUser alloc] initWithObject:userObj];
                         NSArray<RadarGeofence *> *nearbyGeofences = [RadarGeofence geofencesFromObject:nearbyGeofencesObj];
+        
+                        if (user) {
+                            BOOL inGeofences = user.geofences && user.geofences.count;
+                            BOOL atPlace = user.place != nil;
+                            BOOL canExit = inGeofences || atPlace;
+                            [RadarState setCanExit:canExit];
```

Comment:
Pre-existing flag used with `trackingOptions.sync = .stopsAndExits`. Refactored it here.